components/script/dom/urlsearchparams.rs

```diff
@@ -66,13 +83,10 @@ impl URLSearchParams {
                 *query.list.borrow_mut() =
                     form_urlencoded::parse(init_bytes).into_owned().collect();
             },
-            Some(USVStringOrURLSearchParams::URLSearchParams(init)) => {
-                // Step 3.
-                *query.list.borrow_mut() = init.list.borrow().clone();
-            },
-            None => {},
+            _ => {},
```

Comment:
This branch should probably still use `None`, should't it?

Reply:
Ah, yeah! Good catch! I forgot to change it back while testing each implementation, thanks!